docs/site/Components.md

```diff
@@ -0,0 +1,29 @@
+---
+lang: en
+title: 'Components'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Components.html
+---
+
+## Overview
+
+A `Component` is an extensibility class, which allows for creating various types
+of LoopBack 4 extensions.
```

Comment:
Please take a look at the existing definition & description of components in https://loopback.io/doc/en/lb4/Using-components.html. I think we should move some of the content from that page to here.